code/modules/projectiles/projectile.dm

```diff
@@ -678,6 +678,34 @@ So if we are on the 32th absolute pixel coordinate we are on tile 1, but if we a
 		return
 	bullet_act(proj)
 
+//platforms
+/obj/structure/platform/projectile_hit(obj/projectile/proj, cardinal_move, uncrossing)
```

Comment:
that's a straight copy paste of obj projectile hit. Why do we even need that

Reply:
The difference is in the border direction check. As mentioned in the PR, I can just add the check into the parent proc, but I wasn't certain what was best practice.

Otherwise I didn't see a way to call to parent as the parent explicitly has the inverse check, and too many objects use it to justify splitting that out into their own child procs.

Reply:
Is the dir inversed on platform ?

Reply:
What? There's nothing wrong with the dir, it's working as intended. The platform structure represents the separation between a lower and higher level, on the lower level turf.